spec/services/anno_auth_validator_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+require 'webmock/rspec'
+
+RSpec.describe AnnoAuthValidator do
+
+  context 'when authorize is called' do
+    describe 'validate canvas and user' do
+      before(:each) do
+        Rails.application.config.JWT_Password = 'abc123'
+        Rails.application.config.JWT_Canvas_Verification_Url = 'http://testserver.com'
+      end
+
+      it 'check valid JWT' do
+        JWT.stub(:decode).and_return([{"group_id" => "10", "user_id" => "100"}])
+        stub_request(:get, /testserver.com/).
```

Comment:
I feel like this could be extracted into a helper method to clean up the individual tests; you could just pass in the response hash.

Reply:
What about putting the JWT stub in the before method since in this case the response is always the same? One less call each of the tests has to make.